docs/config-schema.md

```diff
@@ -154,6 +154,7 @@ The **nested** `writer` config object has the following fields:
 | `logDatapoints` | no | bool | If the log level is set to `debug` and this is true, all datapoints generated by the agent will be logged. (**default:** `false`) |
 | `logEvents` | no | bool | The analogue of `logDatapoints` for events. (**default:** `false`) |
 | `logTraceSpans` | no | bool | The analogue of `logDatapoints` for trace spans. (**default:** `false`) |
+| `LogTraceSpansFailedToShip` | no | bool | If `true`, traces and spans which weren't successfully received by the backend, will be logged as json. (**default:** `false`) |
```

Comment:
You have to run `make docs` in a linux env to properly generate this, not manually edited.